adventure-crew-game/Assets/Scripts/UI/Dialogue/Dialogue Config.asset

```diff
@@ -15,7 +15,35 @@ MonoBehaviour:
   characters:
   - charName: Griffith
     dialogue:
-    - Hi there
-    - My name is Griffith Berserk
+    - You fought well, but it is time to accept your defeat. You seem resourceful,
```

Comment:
While this is source material appropriate, it is a weird place to start dialogue. It's a little too in media res.